src/test/java/BogdanSandu/tests/Tema6/HerokuAlerts.java

```diff
@@ -0,0 +1,151 @@
+        firstAlertText.getText();
+        System.out.println("Textul alertei nr.1: \"" + firstAlertText.getText() + "\"");
+
+    }
+
+    public static void acceptFirstAlert(){
+        Alert firstAlert = driver.switchTo().alert();
+        firstAlert.accept();
+        System.out.println("Am dat accept la firstAlert!");
+    }
+
+    public static void openSecondAlertButton(){
+        WebElement clickSecondAlertButton =driver.findElement(By.xpath("//input[@id='confirmexample']"));
+        clickSecondAlertButton.click();
+        System.out.println("Am dat click pe secondAlertButton!");
+    }
+
+    public static void secondAlertText(){
+        Alert secondAlert = driver.switchTo().alert();
+        secondAlert.getText();
+        System.out.println("Textul alertei nr.2: \"" + secondAlert.getText() + "\"");
+    }
+
+//    public static void acceptAlert(){
+//        Alert secondAlert = driver.switchTo().alert();
+//        secondAlert.accept();
+//        System.out.println("Am dat acceptat alerta!");
+//    }
+
+    public static void cancelSecondAlert(){
+        Alert secondAlert = driver.switchTo().alert();
+        secondAlert.dismiss();
+        System.out.println("Am dat cancel la secondAlert!");
+    }
+
+    public static void secondAlertAfterCancel(){
+        System.out.println("Textul dupa cancel: \"" + driver.findElement(By.id("confirmexplanation")).getText() + "\"");
+    }
+
+    public static void openThirdAlertButton(){
+        WebElement clickThirdAlertButton = driver.findElement(
+                By.xpath("//input[@id='promptexample']"));
+        clickThirdAlertButton.click();
+        System.out.println("Am dat click pe thirdAlertButton!");
+    }
+
+    public static void alertOfThirdButton(){
+        Alert alert = driver.switchTo().alert();
+        String alertText = alert.getText();
+        System.out.println("Textul alertei nr.3: \"" + alert.getText() + "\"");
+        alert.sendKeys("Text modificat");
+        alert.accept();
+        System.out.println("Am dat accept la alerta!");
+    }
+
+    public static void thirdAlertText(){
+        System.out.println("Textul alertei nr.3: \"" + driver.findElement(By.id("promptexplanation")).getText() + "\"");
+    }
+
+    public static void scrollToSubmitButton() {
+        WebElement submitButton = driver.findElement(By.xpath("//input[@type='submit']"));
+        Actions actions = new Actions(driver);
+        actions.moveToElement(submitButton).build().perform();
+        submitButton.click();
+        System.out.println("Am facut scroll catre submit button");
+    }
+
+    public static void tryToFindElement(){
+        try {
+            WebElement tryFind = driver.findElement(By.id("Boo-hoo"));
+        }catch (NoSuchElementException e){
+            FileUtils.takeScreenshot(driver, "inexistentElement");
+            System.out.println("Nu exista elementul cautat!");
+        }
+    }
+
+    public static void closeBrwoser(){
```

Comment:
No need to declare here one extra closeBrowser() method. You can use the one from BrowserManager. Thanks!